core/src/main/java/com/whylogs/core/ColumnProfile.java

```diff
@@ -39,20 +42,42 @@
   @NonNull private final ItemsSketch<String> frequentItems;
   @NonNull private final HllSketch cardinalityTracker;
 
+  static void initNullCheck() {
+    //  "NULL_STRINGS" env var specifies custom values interpreted as null, e.g. "nil.NaN,nan,null"
+    //  Missing value (actual null object) is always included.
+    if (ColumnProfile.nullStrs == null) {
+      ColumnProfile.initNullCheck(System.getenv("NULL_STRINGS"));
+    }
+  }
+
+  static void initNullCheck(String nullSpec) {
+    //  Customize null values, e.g. "nil.NaN,nan,null"; Missing value (actual null object) is always
+    // included.
+    ColumnProfile.nullStrs = nullSpec == null ? null : Arrays.asList(nullSpec.split(","));
```

Comment:
Use `ImmutableList.of` unless you need this list to be mutable for application logic